charts/pega/values-large.yaml

```diff
@@ -274,9 +274,10 @@ installer:
   # Upgrade specific properties
   upgrade:
     # Type of upgrade
-    # Valid values are 'in-place' , 'out-of-place'
+    # Valid values are 'in-place' , 'zero-downtime' , 'custom' , 'out-of-place-rules' , 'out-of-place-data';
```

Comment:
punctuation tweaks:
# Valid `upgradeType` values are 'in-place' , 'zero-downtime' , 'custom' , 'out-of-place-rules' , 'out-of-place-data' .

and perhaps modify the default value to zero-downtime??